src/app.ts

```diff
@@ -42,8 +41,7 @@ app.set("port", process.env.PORT || 3000);
 app.set("views", path.join(__dirname, "../views"));
 app.set("view engine", "pug");
 app.use(compression());
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.json());
```

Comment:
```suggestion
app.use(express.json());
app.use(express.urlencoded({ extended: true }));
```
Otherwise the login is broken, :)

Reply:
It looks like the property `extended: true` is [default](https://expressjs.com/en/api.html) now.  So we might even be able to just use `app.use(express.urlencoded());` if desired.